### PR TITLE
Everest 623 enable pitr for my sql

### PR DESCRIPTION
**Pull request by solovevayaroslavna**
after design/documentation review: 
pg:

image

image

mysql:

image

image

mongo:

image

image

changes in db-details:
image

### CODE REVIEW

File: apps/everest/src/pages/database-form/steps/pitr/pitr-step.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  // const { dbClusterData } = useDatabasePageDefaultValues(mode);
+  const [pitrEnabled, backupsEnabled, pitrStorageLocation, dbType] = watch([
+    DbWizardFormFields.pitrEnabled,
+    DbWizardFormFields.backupsEnabled,
+    DbWizardFormFields.pitrStorageLocation,
+    DbWizardFormFields.dbType,
+  ]);
+
+  useEffect(() => {
+    if (mode === 'new' && backupStorages?.length > 0) {
```
Comment (fabio-silva) [inline apps/everest/src/pages/database-form/steps/pitr/pitr-step.tsx:42]: let's start with `if( backupStorages?.length > 0)` and do the remaining checks inside

**Review by fabio-silva — CHANGES_REQUESTED** [review]

File: apps/everest/src/pages/database-form/steps/pitr/pitr-step.tsx (hunk 1 above)
Comment (solovevayaroslavna) [reply apps/everest/src/pages/database-form/steps/pitr/pitr-step.tsx]: fixed

File: apps/everest/src/pages/database-form/steps/pitr/pitr-step.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+          sx: { my: 1 },
+        }}
+      />
+      {pitrEnabled && (
+        <AutoCompleteAutoFill
+          name={DbWizardFormFields.pitrStorageLocation}
+          label={StorageLocationMessages.storageLocation.label}
+          loading={isFetching}
+          options={backupStorages}
+          autoCompleteProps={{
```
Comment (filipmikes1) [inline apps/everest/src/pages/database-form/steps/pitr/pitr-step.tsx:110]: You probably don't need to add those props here, they should be already included in the AutoCompleteAutoFill component

**Review by filipmikes1 — CHANGES_REQUESTED** [review]

File: apps/everest/src/pages/database-form/steps/pitr/pitr-step.tsx (hunk 2 above)
Comment (solovevayaroslavna) [reply apps/everest/src/pages/database-form/steps/pitr/pitr-step.tsx:110]: hmm. thanks. i'm going to check!

Comment (solovevayaroslavna) [reply apps/everest/src/pages/database-form/steps/pitr/pitr-step.tsx]: fixed

**Pull request merged**